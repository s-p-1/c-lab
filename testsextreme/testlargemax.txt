disable_output
A1=MAX(A2:ZZZ999)
A2=MAX(A3:ZZZ999)
A3=MAX(A4:ZZZ999)
A4=MAX(A5:ZZZ999)
A5=MAX(A6:ZZZ999)
A6=MAX(A7:ZZZ999)
A7=MAX(A8:ZZZ999)
A8=MAX(A9:ZZZ999)
A9=MAX(A10:ZZZ999)
A10=MAX(A11:ZZZ999)
A11=MAX(A12:ZZZ999)
A12=MAX(A13:ZZZ999)
A13=MAX(A14:ZZZ999)
A14=MAX(A15:ZZZ999)
A15=MAX(A16:ZZZ999)
A16=MAX(A17:ZZZ999)
A17=MAX(A18:ZZZ999)
A18=MAX(A19:ZZZ999)
A19=MAX(A20:ZZZ999)
A20=MAX(A21:ZZZ999)
A21=MAX(A22:ZZZ999)
A22=MAX(A23:ZZZ999)
A23=MAX(A24:ZZZ999)
A24=MAX(A25:ZZZ999)
A25=MAX(A26:ZZZ999)
A26=MAX(A27:ZZZ999)
A27=MAX(A28:ZZZ999)
A28=MAX(A29:ZZZ999)
A29=MAX(A30:ZZZ999)
A30=MAX(A31:ZZZ999)
A31=MAX(A32:ZZZ999)
A32=MAX(A33:ZZZ999)
A33=MAX(A34:ZZZ999)
A34=MAX(A35:ZZZ999)
A35=MAX(A36:ZZZ999)
A36=MAX(A37:ZZZ999)
A37=MAX(A38:ZZZ999)
A38=MAX(A39:ZZZ999)
A39=MAX(A40:ZZZ999)
A40=MAX(A41:ZZZ999)
A41=MAX(A42:ZZZ999)
A42=MAX(A43:ZZZ999)
A43=MAX(A44:ZZZ999)
A44=MAX(A45:ZZZ999)
A45=MAX(A46:ZZZ999)
A46=MAX(A47:ZZZ999)
A47=MAX(A48:ZZZ999)
A48=MAX(A49:ZZZ999)
A49=MAX(A50:ZZZ999)
A50=MAX(A51:ZZZ999)
A999=B1
enable_output
B1=2
A20=5
q

