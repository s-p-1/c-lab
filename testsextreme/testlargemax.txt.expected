              A           B           C           D           E           F           G           H           I           J
  1           0           0           0           0           0           0           0           0           0           0
  2           0           0           0           0           0           0           0           0           0           0
  3           0           0           0           0           0           0           0           0           0           0
  4           0           0           0           0           0           0           0           0           0           0
  5           0           0           0           0           0           0           0           0           0           0
  6           0           0           0           0           0           0           0           0           0           0
  7           0           0           0           0           0           0           0           0           0           0
  8           0           0           0           0           0           0           0           0           0           0
  9           0           0           0           0           0           0           0           0           0           0
 10           0           0           0           0           0           0           0           0           0           0
[0.0] (ok) > [0.0] (ok) > [0.2] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.1] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.1] (ok) > [0.0] (ok) > [0.0] (ok) > [0.1] (ok) > [0.1] (ok) > [0.1] (ok) > [0.1] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.1] (ok) > [0.1] (ok) > [0.1] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.1] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.1] (ok) > [0.1] (ok) > [0.0] (ok) > [0.0] (ok) > [0.1] (ok) > [0.1] (ok) > [0.1] (ok) > [0.0] (ok) > [0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
  1           0           0           0           0           0           0           0           0           0           0
  2           0           0           0           0           0           0           0           0           0           0
  3           0           0           0           0           0           0           0           0           0           0
  4           0           0           0           0           0           0           0           0           0           0
  5           0           0           0           0           0           0           0           0           0           0
  6           0           0           0           0           0           0           0           0           0           0
  7           0           0           0           0           0           0           0           0           0           0
  8           0           0           0           0           0           0           0           0           0           0
  9           0           0           0           0           0           0           0           0           0           0
 10           0           0           0           0           0           0           0           0           0           0
[0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
  1           2           2           0           0           0           0           0           0           0           0
  2           2           0           0           0           0           0           0           0           0           0
  3           2           0           0           0           0           0           0           0           0           0
  4           2           0           0           0           0           0           0           0           0           0
  5           2           0           0           0           0           0           0           0           0           0
  6           2           0           0           0           0           0           0           0           0           0
  7           2           0           0           0           0           0           0           0           0           0
  8           2           0           0           0           0           0           0           0           0           0
  9           2           0           0           0           0           0           0           0           0           0
 10           2           0           0           0           0           0           0           0           0           0
[0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
  1           5           2           0           0           0           0           0           0           0           0
  2           5           0           0           0           0           0           0           0           0           0
  3           5           0           0           0           0           0           0           0           0           0
  4           5           0           0           0           0           0           0           0           0           0
  5           5           0           0           0           0           0           0           0           0           0
  6           5           0           0           0           0           0           0           0           0           0
  7           5           0           0           0           0           0           0           0           0           0
  8           5           0           0           0           0           0           0           0           0           0
  9           5           0           0           0           0           0           0           0           0           0
 10           5           0           0           0           0           0           0           0           0           0
[0.0] (ok) > 